Source/WebDriver/WebDriverService.cpp

```diff
@@ -465,7 +465,7 @@ void WebDriverService::handleMessage(WebSocketMessageHandler::Message&& message,
 
     auto parsedMessageValue = JSON::Value::parseJSON(String::fromUTF8(message.payload.data()));
     if (!parsedMessageValue) {
-        RELEASE_LOG(WebDriverBiDi, "WebDriver handle Message: Failed to parse incoming message");
+        RELEASE_LOG(WebDriverBiDi, "WebDriver handle BiDi message: Failed to parse incoming message");
```

Comment:
Nit: prefer to use the method name here:

`RELEASE_LOG(WebDriverBiDi, "WebDriverService::handleMessage() failed to parse BiDi message as JSON");`